src/machine/Supplies.java

```diff
@@ -0,0 +1,70 @@
+package machine;
+
+public class Supplies {
```

Comment:
minor: better to rename it to `CoffeeMachineSupplies` so all class names start with `Coffee` ☕